scene/gui/flow_container.h

```diff
@@ -57,6 +57,10 @@ class FlowContainer : public Container {
 
 	void _resort();
 
+	Vector<int> cached_children_pos;
+	Vector<int> cached_children_idx;
+	Vector<Node *> cached_visible_children;
```

Comment:
this is not used, is it?

Reply:
No, its a remnant from early experiments.